library/Stratosphere/Types.hs

```diff
@@ -1,19 +1,170 @@
-{-# LANGUAGE DeriveAnyClass #-}
-{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveAnyClass    #-}
+{-# LANGUAGE DeriveGeneric     #-}
 {-# LANGUAGE OverloadedStrings #-}
 
 -- | Module for hand-written types that are used in generated modules.
 
 module Stratosphere.Types
-  ( CannedACL (..)
+  ( EnabledState (..)
+  , AuthorizationType (..)
+  , HttpMethod (..)
+  , LoggingLevel (..)
+  , ApiBackendType (..)
+  , Period (..)
+  , AttributeType (..)
+  , KeyType (..)
+  , ProjectionType (..)
+  , StreamViewType (..)
+  , SNSProtocol (..)
+  , Runtime (..)
+  , CannedACL (..)
   , KinesisFirehoseS3CompressionFormat(..)
   , KinesisFirehoseElasticsearchS3BackupMode(..)
   , KinesisFirehoseNoEncryptionConfig(..)
   ) where
 
-import Data.Aeson
-import Data.Text (unpack)
-import GHC.Generics
+import           Data.Aeson
+import           Data.Text    (unpack)
+import           GHC.Generics
+
+
+data EnabledState
```

Comment:
These new types are great! I wonder if there are going to be any collisions with other libraries. I guess we can change it if someone complains.